schemas/BaseElements.xsd

```diff
@@ -3221,6 +3221,14 @@
 					<xs:attribute name="dataSource" type="DataSource"/>
 				</xs:complexType>
 			</xs:element>
+			<xs:element name="PackagedTerminalAirConditionerHeating">
```

Comment:
@shorowit Is this looking good? `HeatingSystemType` listed elements instead of enumerations, another choice is to use `Other` with extension describing PTAC heating.

Reply:
@nmerket What would you like to do? Better to create new PTACheating element or to use Other element?

Reply:
This looks good.